### ISSUE TITLE
concerning reproducing the result of table 3 

### ISSUE THREAD

**Issue by Daniellli** [SimingYan/IAE#11]
i load the  [pretrained model ](https://drive.google.com/drive/folders/1XX72FsbI56_vMBax7BZSiGVFuCOPffHC) and finetune on the votenet using your [codebase](https://github.com/SimingYan/votenet). However,  the performance in Table 3 can not be reproduced.   
the  finetune script  i use is 
```
python train.py \
--dataset scannet --log_dir log_scannet \
--num_point 40000 --no_height \
--pre_checkpoint_path=~/pretrained_models/scannet.pt \
--batch_size=16
```

the train script i use is 

```
python train.py --dataset scannet --log_dir log_scannet0 --num_point 40000 --no_height --batch_size=16 
```

and the results i got is:
<!DOCTYPE html><p cid="n361" mdtype="paragraph" class="md-end-block md-p" style="box-sizing: border-box; line-height: inherit; orphans: 4; margin: 0.8em 0px; white-space: pre-wrap; position: relative; caret-color: rgb(51, 51, 51); color: rgb(51, 51, 51); font-family: &quot;Open Sans&quot;, &quot;Clear Sans&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, &quot;Segoe UI Emoji&quot;, sans-serif; font-size: 16px; font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; text-align: start; text-indent: 0px; text-transform: none; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration: none;"><br class="Apple-interchange-newline"></p><figure class="md-table-fig" cid="n365" mdtype="table" style="box-sizing: border-box; margin: 1.2em 0px; overflow-x: auto; max-width: calc(100% + 16px); padding: 0px; cursor: default; caret-color: rgb(51, 51, 51); color: rgb(51, 51, 51); font-family: &quot;Open Sans&quot;, &quot;Clear Sans&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, &quot;Segoe UI Emoji&quot;, sans-serif; font-size: 16px; font-style: normal; font-variant-caps: normal; font-weight: normal; letter-spacing: normal; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration: none;">

  | mAP@0.25 | mAP@0.50
-- | -- | --
load the pretrain model | 58.79 | 35.26
train from scrach | 55.61 | 32.92
  |   |  

</figure>

Is  anything wong?  

appreciate your help.

**Comment by SimingYan** [comment]
Hi,

Could you please try to reduce the batch_size to 8?  The batch size matters in this case.

Also, if you are familiar with votenet and their original repo, you will find votenet training is very unstable. Other self-supervised learning methods also suffer from this issue. Therefore, we are extending our work to a better backbone.
